apps/src/templates/instructions/TopInstructions.jsx

```diff
@@ -404,6 +423,10 @@ class TopInstructions extends Component {
 
   handleTeacherOnlyTabClick = () => {
     this.setState({tabSelected: TabType.TEACHER_ONLY}, this.scrollToTopOfTab);
+    firehoseClient.putRecord({
+      study: 'teacher-only-tab',
+      event: 'click'
+    });
```

Comment:
We might want this info more tightly connected in the events table.  Maybe the study name should be consistent across all three?  Something like `top-instructions` and then the events could be `teacher-only-tab-click`, `collapse-instructions`,  and `expand-instructions`.

Reply:
Sounds good to me. I don't know how we normally set these things up.